lib/modules/datasource/metadata.spec.ts

```diff
@@ -10,9 +11,17 @@ import {
 import { NpmDatasource } from './npm';
 import { PypiDatasource } from './pypi';
 import type { ReleaseResult } from './types';
-import { partial } from '~test/util';
+import { hostRules, partial } from '~test/util';
```

Comment:
```suggestion
import { hostRules } from '~test/host-rules';
import { partial } from '~test/util';
```

https://github.com/renovatebot/renovate/blob/12ea7e1474d54351e4a4e636c18d27d6d1030076/test/host-rules.ts#L6